scripts/install_backend_python_libs.py

```diff
@@ -64,11 +85,12 @@ def _get_requirements_file_contents():
             if l.startswith('#') or len(l) == 0:
                 continue
             library_name_and_version_string = l.split(' ')[0].split('==')
-            # Libraries with different case are not different: e.g
-            # 'Flask' == 'flask'. Therefore, we normalize all library names
-            #  in order to compare libraries without ambiguities.
+            # Libraries with different case are considered equivalent libraries:
+            # e.g 'Flask' is the same library as 'flask'. Therefore, we
+            # normalize all library names in order to compare libraries without
+            # ambiguities.
             library_name = (
-                _normalize_string_that_contains_python_library_name(
+                _normalize_python_library_name(
```

Comment:
So this is a normalized_library_name, not just a library_name. (Carry the concepts along throughout the code.)

Reply:
Done.